### PR TITLE
migrate existing cemu and all future cemu to toolslocation

### PR DESCRIPTION
**Pull request by AeliusSaionji**

### CODE REVIEW

File: cemu/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -4,22 +4,27 @@ $packageName = 'cemu'
...
-  url          = $url32
+  url            = $url32
   checksum       = $checksum32
   checksumType   = 'sha256'
   unzipLocation  = $toolsPath
 }
 
+If (Get-Item "$env:ChocolateyInstall\lib-bkp\cemu\tools\cemu\Cemu.exe" -ErrorAction SilentlyContinue) {
+	Write-Output "Migrating cemu location to $cemu_folder"
+	Move-Item "$env:ChocolateyInstall\lib-bkp\cemu\tools\cemu" "$(Get-ToolsLocation)" -Force
```
Comment (AeliusSaionji) [inline cemu/tools/chocolateyinstall.ps1:19]: chocolatey moves the entire folder to lib-bkp after `beforemodify.ps1` runs, but before `chocolateyinstall.ps1`. The `beforemodify.ps1` from the previous version is used- it runs before the update is even downloaded, so we cannot put any of the  migration code in it.

File: cemu/tools/chocolateyinstall.ps1 [hunk 2]
```diff
@@ -4,22 +4,27 @@ $packageName = 'cemu'
...
+	Remove-Item "$env:ChocolateyInstall\lib\cemu\tools\cemu" -Recurse -Force -ErrorAction SilentlyContinue
```
Comment (AeliusSaionji) [inline cemu/tools/chocolateyinstall.ps1:20]: Before running `chocolateyinstall.ps1`, chocolatey copies _known files_* back from lib-bkp to lib. This is incomplete and not worth saving in this context.

*: by known files I mean whatever files in the cemu zip that were extracted by chocolatey. Any files generated by cemu or the user are not included here.

**Review by SebastianK90 — COMMENTED** [review]
Sorry I have absolutely no time at the moment

**Comment by AeliusSaionji** [comment]
No problem, take your time. I did test this pretty thoroughly, but another set of eyes should look at it 😄

**Pull request merged**